ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml

```diff
@@ -148,6 +148,9 @@ tests:
     test:
     - as: openshift-e2e-test
       commands: bash ./hack/e2e.sh
+      dependencies:
+      - env: CINCINNATI_IMAGE
+        name: deploy
```

Comment:
Hmm, we [don't have any release branch testing after 4.8][1]?  Do we expect to just have a master branch for Cincinnati, without ever cutting release branches?  I'm trying to figure out if something like this change will ever need to be cloned back to release configs for earlier branches.

[1]: https://github.com/openshift/release/tree/3829ff145cb46bac790c7fdbec873c0eeab5cfb1/ci-operator/config/openshift/cincinnati

Reply:
I think its easier to finalize master changes before we cut a 4.9 branch (this change should be cloned to release-4.x branches later on to ensure stability, yes)